docker-compose.dev.yml

```diff
@@ -49,7 +49,7 @@ services:
       - CELERY_BROKER_URL=redis://redis:6379
       - CELERY_RESULT_BACKEND=redis://redis:6379
     image: python:3.9
-    command: /bin/bash -c 'python -m venv .ibutsu_env && source .ibutsu_env/bin/activate && pip install -U pip setuptools wheel && pip install -r requirements.txt && ./celery_worker.sh'
+    command: ./worker-entrypoint.sh
```

Comment:
why not declare that either worker depends on backend or backend depends on worker?

Reply:
if you mean `depends_on:` keyword, it's already there. The problem is how it works https://docs.docker.com/compose/compose-file/compose-file-v3/#depends_on

```
depends_on does not wait for db and redis to be “ready” before starting web - only until they have been started. If you need to wait for a service to be ready, see [Controlling startup order](https://docs.docker.com/compose/startup-order/) for more on this problem and strategies for solving it.
```

Such wait for script is the recommended solution https://docs.docker.com/compose/startup-order/